src/scripts.js

```diff
@@ -31,17 +31,21 @@ let currentView;
 
 //event listeners
 recipeSection.addEventListener('click', createRecipeModal);
+recipeSection.addEventListener('keypress', createRecipeModal);
```

Comment:
@LordSchwifty Does this allow _any_ key to open the modal? If so, do we want that functionality?

cc @jsahim @qrispi